docs/playground/ko/TypeScript/Language/Type Guards.ts

```diff
@@ -0,0 +1,91 @@
+// 타입 가드는 코드를 통해 코드 흐름 분석에 영향을 주는 용어입니다.
+// TypeScript는 코드 흐름에 영향을 미치기 위해
+// 런타임에서 오브젝트가 유효한지 판단하는 기존의 JavaScript 동작을 사용합니다.
```

Comment:
```suggestion
// TypeScript는 런타임에서 오브젝트가 유효한지 판단하는 
// 기존의 JavaScript 동작을 이용해서 코드 흐름에 영향을 줄 수 있습니다.
```
[제안]
어색한 문장 위치를 수정해봤습니다.